megaavr/cores/dxcore/low_power.cpp

```diff
@@ -0,0 +1,32 @@
+#include "Arduino.h"
+
+// MALS-89
+void setSleepMode(uint8_t sleepMode) {
```

Comment:
This isn't the same signature as in the API specification. This should take an `SLPCTRL_SMODE_t` not a generic uint_8.